### ISSUE TITLE
'Automatic Power Off' should not happen if switch is already off

### ISSUE THREAD

**Issue by vogler** [jneilliii/OctoPrint-TasmotaMQTT#43]
I get 'Automatic Power Off' messages every 5mins despite the switch being turned off. I was just setting up OctoPrint and noticed it playing around while the printer is off.

![image](https://user-images.githubusercontent.com/493741/90297399-9c772e00-de8e-11ea-9248-bf5c07db81e3.png)
Settings:
![image](https://user-images.githubusercontent.com/493741/90297444-c3356480-de8e-11ea-9315-16a35c66a6f4.png)

**Comment by jneilliii** [comment]
Yeah, that makes sense for having just one relay, but with multiple relays that this plugin supports this will be a little more difficult to check against because the idle timer is for all. I'll definitely look into it when I get a chance. It should only start the idle timer once though. Once it times out it should go away unless I forgot to include something when I migrated this feature over from my other plugins.

**Comment by vogler** [comment]
Maybe I don't understand the problem with multiple relays, but wouldn't it be enough to not start the timer if all relays are off?
Something like
~~~python
  if self.powerOffWhenIdle and any(map(lambda r: r["currentstate"] == "ON", self._settings.get(["arrRelays"]))):
~~~
instead of https://github.com/jneilliii/OctoPrint-TasmotaMQTT/blob/0dcfade79eac7332825bc5ff415b720eac94775e/octoprint_tasmota_mqtt/__init__.py#L406

**Comment by jneilliii** [comment]
Thanks for that code snippet. Would be glad to incorporate that into the plugin once I've had time to test. You may notice I'm not really a great python programmer.

**Comment by jneilliii** [comment]
change merged and officially released as 0.3.7, thanks for the PR @vogler.
